mu-plugins/wp-ignition/wp-ignition.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Plugin Name:       WP Ignition
+ * Description:       Adds Laravel Ignition to WordPress to aid with debugging.
+ * Version:           1.0.0
+ * Author:            10up
+ * Author URI:        https://10up.com/
+ * License:           GPL v2 or later
+ * License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+ * Text Domain:       wp-ignition
+ * Update URI:        https://yourwebsite.com/wp-ignition
+ * Requires at least: 5.0
+ * Requires PHP:      8.0
+ *
+ * @package WP_Ignition
+ */
+
+require __DIR__ . '/vendor/autoload.php';
+
+// Register Ignition
+\Spatie\Ignition\Ignition::make()
+	->setTheme( 'dark' )
+	->shouldDisplayException( 'production' !== wp_get_environment_type() && true === WP_DEBUG )
```

Comment:
**question:** is this needed if this file is never loaded because of https://github.com/10up/wp-scaffold/pull/256/files#diff-4ea83a254277eb00b13e53d487e576481ad61df4a542af71e5ed0df5df2f9333R19

Reply:
It's mostly there as a backup in case someone _does_ deploy it, or if they pull it into a pre-existing project.